### ISSUE TITLE
bintrayUpload fails due to lacking license in generated POM file

### ISSUE THREAD

**Issue by kuniss** [oehme/sobula#4]
Hi @oehme 
I use your Gradle plug-in to publish to Bintray and MavenCentral for a license not supported by your plug-in. In particular it's the LGPL license.
To get it published to Bintray, I added the following line to my Gradle build script which gets it uploaded to Bintray:

```
bintray.pkg.licenses = ["LGPL-3.0"]
```

However the POM generated does not get a license XML node from that setting letting the upload to MavenCentral fail:

```
Could not sync 'kuniss/maven/de.grammarcraft.scala.flow/0.1.1' to Maven Central: HTTP/1.1 400 Bad Request [messages:[Invalid POM: /de/grammarcraft/de.grammarcraft.scala.flow/0.1.1/de.grammarcraft.scala.flow-0.1.1.pom: License information missing, Dropping existing partial staging repository.], status:Validation Failed]
```

Do I miss something? Or does it not work at all if the license is not supported by your plug-in?

Thanks, for any hint.
Denis

**Comment by oehme** [comment]
Hi Denis,

unfortunately you also have to adjust the generated pom.xml using some
Groovy XML processing, which is not so great.

I think it would be much more straightforward (and helpful for others) if
you open a Pull Request with more auto-detected licenses =)

Cheers,
Stefan

2015-12-10 21:39 GMT+01:00 Denis Kuniß notifications@github.com:

**Comment by kuniss** [comment]
Well, I will try it.
Denis

Am 10.12.2015 um 22:07 schrieb Stefan Oehme:

**Comment by oehme** [comment]
Fixed, see #5
